vcs/local.go

```diff
@@ -0,0 +1,44 @@
+package vcs
+
+import (
+	"fmt"
+)
+
+func init() {
+	Register(newLocal, "local")
+}
+
+type LocalDriver struct{}
+
+func newLocal(b []byte) (Driver, error) {
+	return &LocalDriver{}, nil
+}
+
+func (g *LocalDriver) HeadRev(dir string) (string, error) {
+	return "n/a", nil
+}
+
+func (g *LocalDriver) Pull(dir string) (string, error) {
+	return g.HeadRev(dir)
+}
+
+func (g *LocalDriver) Clone(dir, url string) (string, error) {
+	// For local driver Clone() is only called when the directory
+	// pointed by url is not found.
+	err := fmt.Errorf("Location %s not found.", url)
+	fmt.Print(err)
```

Comment:
All the other drivers use `log` to report the actual error. `fmt.Print` is not only going to give an inconsistent log line, but it also won't print a \n at the end of the line.